GitUI/UserControls/FileStatusList.cs

```diff
@@ -1200,6 +1205,26 @@ private void FileStatusListView_KeyDown(object sender, KeyEventArgs e)
             }
         }
 
+        private bool _mouseEntered;
```

Comment:
whoops, this should probably be at the top

Reply:
yes, please

Reply:
> 
> 
> To confirm - if use tab to cycle between controls, we don't get flickering, do we?

We don't with this fix and we didn't as when you'd tab around the first item gets selected so even if it used to flicker _technically_ it wasn't visible as it'd `flicker` between the same item.